api/auth_token.go

```diff
@@ -15,13 +15,17 @@ func (a *Auth) Token() *TokenAuth {
 }
 
 func (c *TokenAuth) Create(opts *TokenCreateRequest) (*Secret, error) {
+	ctx, cancelFunc := context.WithCancel(context.Background())
+	defer cancelFunc()
+	return c.CreateContext(ctx, opts)
+}
+
+func (c *TokenAuth) CreateContext(ctx context.Context, opts *TokenCreateRequest) (*Secret, error) {
```

Comment:
A tiny nit here, can you update these to `<method>WithContext(...)` to match the existing pattern  `RawRequestWithContext`.

Reply:
No probled

Reply:
Done